src/Libraries/Microsoft.Extensions.AI.AzureAIInference/AzureAIInferenceChatClient.cs

```diff
@@ -51,7 +51,7 @@ public AzureAIInferenceChatClient(ChatCompletionsClient chatCompletionsClient, s
     }
 
     /// <summary>Gets or sets <see cref="JsonSerializerOptions"/> to use for any serialization activities related to tool call arguments and results.</summary>
-    public JsonSerializerOptions? ToolCallJsonSerializerOptions { get; set; }
+    public JsonSerializerOptions ToolCallJsonSerializerOptions { get; set; } = AIJsonUtilities.DefaultOptions;
```

Comment:
If we're going to depend on this being non-null, we should validate in the setter that the provided value is non-null.

Reply:
Done.